webapp.py

```diff
@@ -264,14 +264,17 @@ def log_error(self, format, *args):
         logger.error(message)
 
 
-def run_server(port, settings, keyfile=None, certfile=None):
+def create_server(port, settings, keyfile=None, certfile=None):
     """
-    Run the server until a shutdown is requested.
+    Create the server.
 
     :param int port: A port number to listen to.
     :param str settings: Path to settings file.
     :param str keyfile: Optional path to a PEM key file to run in https mode.
     :param str certfile:  Optional path to a PEM certificate file to run in https mode.
+
```

Comment:
This was refactored a bit so we can easily create a server from `test_integration.py`.